.github/workflows/gh-build.yml

```diff
@@ -54,29 +61,26 @@ jobs:
 
       - name: Build docker image
         run: |
-          echo BUILD_TARGET: ${{ inputs.build-target }}
-          echo USE_CUDA: ${{ env.USE_CUDA }}
-
           IMAGE_TAG=${{ env.IMAGE_NAME }}:${{ inputs.sha }}
 
+          chmod +x continuous_integration/build-docker-image
```

Comment:
Should the mode of the file be changed in the repo instead of here?

Reply:
Yes.